### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by FlorRodriguezA**
## Summary
- Responsive card

## Screenshot
Captura de pantalla de 2021-03-02 19-43-36
Captura de pantalla de 2021-03-02 19-44-02

## Trello Card
https://trello.com/c/M0QUonOW

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -39,5 +39,6 @@ <h3 class="book-year">Año de publicación: </h3>
         </div>
       </div>
    </div>
+   </div>
```
Comment (mcallegari10) [inline maquetado/index.html]: I think this isn't necessary

File: maquetado/scss/detail.scss [hunk 2]
```diff
@@ -116,3 +121,86 @@
   font-size: 20px;
   color: $gray;  
 }
+
+@media only screen and (max-width: 1024px) {
+  [class='cover'] {
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: Use the class notation 😝 

```css
.cover {
  ...
}
```

File: maquetado/scss/detail.scss [hunk 3]
```diff
@@ -116,3 +121,86 @@
...
+    height: 667px;
+    width: 100vw;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: I think this isn't necessary

File: maquetado/scss/detail.scss [hunk 4]
```diff
@@ -116,3 +121,86 @@
...
+    font-size: 14px;
+  }
+
+  [class='book-year'] {
+    font-size: 14px;
+  }
+
+  [class='data'] {
+    font-size: 14px;
+  }
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: Maybe you can use `nth-child` so you won't need to repeat so much 😄

File: maquetado/scss/detail.scss [hunk 5]
```diff
@@ -116,3 +121,86 @@
...
+}
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: EOF line

File: maquetado/scss/detail.scss [hunk 6]
```diff
@@ -116,3 +121,86 @@
...
   color: $gray;  
 }
+
+@media only screen and (max-width: 1024px) {
+  [class='cover'] {
+    height: 667px;
+    width: 100vw;
+  }
+  [class='container'] {
+    width: 100%;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: This is already in the class, no need to add 😄

File: maquetado/scss/detail.scss [hunk 7]
```diff
@@ -116,3 +121,86 @@
...
+    max-width: 376px;
+    height: 667px;
+    display: flex;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: This is already in the class

File: maquetado/scss/detail.scss [hunk 8]
```diff
@@ -116,3 +121,86 @@
...
+    flex-flow: column-reverse;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: `reverse` is what you need? 🤔

File: maquetado/scss/detail.scss [hunk 9]
```diff
@@ -116,3 +121,86 @@
...
+  [class='cover-container'] {
+    &::after {
+      /*No toma el primer width*/
+      width: $widthBadgeCellphone;
+      height: 119px;
+      width: 66px;
+      /*Se que no esta bien posicionar así pero no me
+      quedaba en la ubicación*/
+      left: 160px;
+      bottom: 200px;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: It's OK. Maybe you can use percentages or variables for the book sizes

File: maquetado/scss/detail.scss [hunk 10]
```diff
@@ -1,6 +1,11 @@
+$widthBadgeDesktop: 95px;
+$widthBadgeCellphone: calc(var($widthBadgeDesktop) * 70%);
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: I think this won't work. The proper way to use SASS variables in `calc` functions is:

```scss
$widthBadgeCellphone: calc(#{$widthBadgeDesktop} * 70%);
```

Also, available for other CSS functions if you need to use SASS variables 😉

File: maquetado/scss/detail.scss [hunk 11]
```diff
@@ -1,6 +1,11 @@
...
+
 .cover {
   width: 100vw;
   background-color: $wild-sand;
+  height: 768px;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: I think this shouldn't be here or this size 🤔

**Review by mcallegari10 — CHANGES_REQUESTED** [review]
I think you are missing the meta tag in the index.html. Check this https://css-tricks.com/snippets/html/responsive-meta-tag/

**Comment by FlorRodriguezA** [comment]
Summary
I finished the Responsive card

Screenshot
Captura de pantalla de 2021-03-03 14-51-47

Trello Card
https://trello.com/c/M0QUonOW

File: maquetado/scss/detail.scss [hunk 12]
```diff
@@ -1,6 +1,12 @@
+$widthBadgeDesktop: 95px;
+$widthBadgeCellphone: calc(#{$widthBadgeDesktop} * 0.7);
+
 .cover {
   width: 100vw;
   background-color: $wild-sand;
+  .container {
+    max-width: 300px;
+  }
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: Delete this 😄

File: maquetado/scss/detail.scss [hunk 13]
```diff
@@ -97,22 +103,72 @@
...
+
+  .book {
+    height: 570px;
+    width: 336px;
+    align-content: center;
+    justify-content: center;
+  }
+
+  .cover-container {
+    &::after {
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]:
```suggestion
  .cover-container::after {
```

**Review by mcallegari10 — APPROVED** [review]
Just those 2 details 😄 . Great job!

**Comment by FlorRodriguezA** [comment]
Summary
I finished the css grid card

Screenshot
Captura de pantalla de 2021-03-05 11-29-32
Captura de pantalla de 2021-03-05 11-29-44
Captura de pantalla de 2021-03-05 11-29-54


Trello Card
https://trello.com/c/3ePt3n4T

File: maquetado/scss/detail.scss [hunk 14]
```diff
@@ -33,44 +44,57 @@
...
 
 .book-cover {
-  margin: 24px 52px 24px 24px;
   height: 368px;
   width: 261px;
   position: relative;
 }
 
+
+
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]:
```suggestion
```

File: maquetado/scss/detail.scss [hunk 15]
```diff
@@ -10,6 +14,12 @@
   max-width: 900px;
   width: 100%;
   justify-content: center;
+  display: grid;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss:17]: Remember to delete the `display: flex;` above

File: maquetado/scss/detail.scss [hunk 16]
```diff
@@ -97,22 +128,171 @@
...
 }
 
-.book-year {
-  font-size: 20px;
-  font-weight: 700;
+@media only screen and (max-width: 1366px) {
+  .container {
+    max-width: 1024px;
+    height: 768px;
+    display: grid;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: No need for this 😄

File: maquetado/scss/detail.scss [hunk 17]
```diff
@@ -97,22 +128,171 @@
...
+  color: $gray;  
 }
 
-.book-year {
-  font-size: 20px;
-  font-weight: 700;
+@media only screen and (max-width: 1366px) {
+  .container {
+    max-width: 1024px;
+    height: 768px;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: You can leave it in `auto` I think

File: maquetado/scss/detail.scss [hunk 18]
```diff
@@ -97,22 +128,171 @@
...
+.data {
   font-size: 20px;
-  font-weight: 700;
+  color: $gray;  
 }
 
-.book-year {
-  font-size: 20px;
-  font-weight: 700;
+@media only screen and (max-width: 1366px) {
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: Remember to move the breakpoint to a variable 😄

File: maquetado/scss/detail.scss [hunk 19]
```diff
@@ -97,22 +128,171 @@
...
+
+  .book-cover {
+    margin: 0 36px 56px 32px;
+  }
+  
+  .book-title {
+    grid-area: header;
+    justify-self: center;
+    margin-top: 22px;
+    align-self: flex-start;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: I think this is not necessary or check that flex, remember this is grid 😝

File: maquetado/scss/detail.scss [hunk 20]
```diff
@@ -97,22 +128,171 @@
...
+
+  .detail-label, .data{
+    font-size: 24px;
+  }
 }
 
-.data {
-  font-size: 20px;
-  color: $gray;  
+@media only screen and (max-width: 1024px) {
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: Move breakpoint to a variable 😄

File: maquetado/scss/detail.scss [hunk 21]
```diff
@@ -97,22 +128,171 @@
...
+  }
+
+  .back {    
+    font-size: 18px; 
+    grid-area: footer;
+    justify-self: center;
+  }
+
+  .book {
+    height: 570px;
```
Comment (mcallegari10) [inline maquetado/scss/detail.scss]: No need for the height

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**